core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -294,11 +292,11 @@ describe('Creator dashboard controller', () => {
                   .createFromBackendDict(feedbackThread))),
             createdSuggestionsList: (
               dashboardData.created_suggestions_list.map(
-                suggestionDict => suggestionObjectFactory
+                suggestionDict => Suggestion
```

Comment:
Please do it like this, the same in other places.
```suggestion
                suggestionDict => Suggestion.createFromBackendDict(
                  suggestionDict
                ))),
```

Reply:
Done!